app/code/Magento/Ui/view/base/web/js/modal/modal-component.js

```diff
@@ -361,7 +361,44 @@ define([
 
             if (this.valid) {
                 this.closeModal();
+            } else {
+                this._focusInvalid(this.elems());
+            }
+        },
+
+        /**
+         * Focus to first invalid element
+         *
+         * @param {HTMLElement} elems
+         * @param {boolean} firstRun
+         * @returns {boolean}
+         */
+        _focusInvalid: function (elems, firstRun = true) {
```

Comment:
Hi @edenduong, thanks for collaboration. 

Please rename this function, the name with underscore is deprecated.
Please remove the set of default value from the function declaration. It's ES6 feature, but currently, we support just ES5